src/annotator/components/NotebookModal.js

```diff
@@ -0,0 +1,96 @@
+import { createSidebarConfig } from '../config/sidebar';
+import { useEffect, useRef, useState } from 'preact/hooks';
+import classnames from 'classnames/bind';
```

Comment:
I've never seen `classnames/bind` before. Was there a specific reason for using it here? The usage below looks to me like it would work the same with the regular "classnames" import.

Reply:
Good catch! This is unnecessary (I was playing around and I forgot to clean up).